README.md

```diff
@@ -43,44 +42,56 @@ How about:
 ![animated svg of the tldr client displaying the tar command](images/tldr.svg)
 
 This repository is just that: an ever-growing collection of examples
-for the most common UNIX, Linux, macOS, SunOS and Windows command-line tools.
+for the most common UNIX, Linux, macOS, SunOS, Android and Windows command-line tools.
 
 ## How do I use it?
 
 A popular and convenient way to access these pages on your computer
 is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-client),
 which is supported by the tldr-pages project maintainers:
 
-    npm install -g tldr
+```sh
+npm install -g tldr
+```
+
+Alternatively, you can also use the [Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
 
-You can also use the fully-featured [official Python client](https://github.com/tldr-pages/tldr-python-client) which can be installed via `pip3`.
+```sh
+pip3 install tldr
+```
 
-    pip3 install tldr
+Then you have direct access to simplified, easy-to-read help for commands, such as tar,
+accessible through typing `tldr tar` instead of the standard `man tar`.
 
-That way you can write `tldr tar` in the terminal to show the tldr page for `tar`,
-just like you would write `man tar` to show its manpage.
+If you just want to browse without an internet connection or installing software,
```

Comment:
```suggestion
If you just want to browse in the future, without an internet connection or installing software,
```
Open to other ideas here, the current sentence implies that the PDF can be downloaded without an internet connection.

Reply:
My suggestion.
```suggestion
If you just want to browse without installing a tldr client,
```

Reply:
```suggestion
If you want an offline version without installing any software,
```
I guess I'll join in too.